termsaverlib/screen/__init__.py

```diff
@@ -104,6 +110,10 @@ def get_available_screens():
 
     return screens
 
+def absoluteFilePaths(directory):
```

Comment:
your code using this function is commented out... is this used?

Reply:
Good catch, no, it's no longer used. I was trying multiple methods of getting the file path without modifying the existing code too much. I should have cleaned this up.

Reply:
Cleaned in new update.